docs/en/installation.rst

```diff
@@ -30,36 +30,41 @@ ESP-IDF and Tools Installation
 -------------------------------
 .. _esp-idf-tools-installation:
 
-The Espressif-IDE provides a tool manager to install and manage ESP-IDF and related tools. Please follow the instructions below for installing ESP-IDF in the Espressif-IDE.
+The Espressif-IDE provides a Tool Manager view that allows you to activate available ESP-IDF versions and manage related tools. To install the ESP-IDF and its tools, first download the Espressif Installation Manager (EIM) from the following `link <https://dl.espressif.com/dl/eim/>`_.
+
+
+After downloading and launching EIM, follow the on-screen instructions to install the required ESP-IDF version and associated tools. Once the installation is complete, you can use the ESP-IDF Manager within the IDE to activate the installed ESP-IDF version in the workspace.
```

Comment:
Can you link EIM documentation here for installation instructions- https://docs.espressif.com/projects/idf-im-ui/en/latest/